geomstats/geometry/general_linear.py

```diff
@@ -111,16 +139,18 @@ def exp(cls, tangent_vec, base_point=None):
     @classmethod
     def log(cls, point, base_point=None):
         r"""
-        Calculate a left-invariant vector field bringing base_point to point.
+        Compute a left-invariant vector field bringing base_point to point.
 
         The output is a vector of the tangent space at base_point, so not a Lie
         algebra element if it is not the identity.
 
         Parameters
         ----------
         point : array-like, shape=[..., n, n]
+            Point.
         base_point : array-like, shape=[..., n, n]
-            Defaults to identity.
+            Base point.
+            Optional, default to identity if None.
```

Comment:
I think this should be either default: identity or defaults to identity (with the s)

Reply:
I checked with a native, `defaults to` works, I have updated all the files.